cromshell

```diff
@@ -711,21 +711,23 @@ function slim-metadata()
 # Get the status of the given job and how many times it was run
 function execution-status-count() 
 {
-  assertCanCommunicateWithServer $2
+  assertCanCommunicateWithServer "$2"
   turtle
   f=$( makeTemp )
-  curl --connect-timeout $CURL_CONNECT_TIMEOUT --max-time $CURL_MAX_TIMEOUT --compressed -s ${2}/api/workflows/v1/$1/metadata?${CROMWELL_METADATA_PARAMETERS} > $f
-  [[ $? -ne 0 ]] && error "Could not connect to Cromwell server." && return 9
+  if ! curl --connect-timeout "$CURL_CONNECT_TIMEOUT" --max-time "$CURL_MAX_TIMEOUT" --compressed -s "$2/api/workflows/v1/$1/metadata?includeKey=executionStatus&includeKey=backendStatus&expandSubWorkflows=true" > "$f" ; then
```

Comment:
Can you revert the syntax to the 2 line check (again for consistency with the rest of the script)?